tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27, pycodestyle, pylint
+envlist = py37, pycodestyle, pylint
```

Comment:
python client are for python2 and python3 tox could test both

Reply:
Python2 is end of life very soon. So we should only support Python 3 going on...